packages/jest-util/src/fake_timers.js

```diff
@@ -225,11 +225,37 @@ export default class FakeTimers {
     }
 
     if (i === this._maxLoops) {
+      const timers = Object.assign({}, this._timers);
       throw new Error(
         'Ran ' +
           this._maxLoops +
           ' timers, and there are still more! ' +
-          "Assuming we've hit an infinite recursion and bailing out...",
+          "Assuming we've hit an infinite recursion and bailing out." +
+          'The following timers are still pending:' +
+          String(
```

Comment:
this is unnecessary, you should do a `reduce` on the array returned by `map` instead

Reply:
Or just `.join`

Reply:
thx, i will do that today